sunbeam/CMakeLists.txt

```diff
@@ -15,7 +15,8 @@ add_library( sunbeam SHARED eclipse_state.cpp
                             schedule.cpp
                             sunbeam.cpp
                             table_manager.cpp
-                            well.cpp )
+                            well.cppy
+                            groupTree.cpp )
```

Comment:
You can see that `table_manager.cpp` is written snake case, so `groupTree.cpp` should be `group_tree.cpp`.

What is `well.cppy`?